glossary/BEM.md

```diff
@@ -0,0 +1,16 @@
+# BEM
+
+[BEM](https://en.bem.info/) is a methodology and libraries developed and used at Yandex for building user interfaces.
+
+Key concepts of methodology
+ * Block: logically and functionally independent page component, encapsulates behavior (JavaScript), templates, styles (CSS), and other implementation technologies
+ * Element: a constituent part of a block that can’t be used outside of it (For example, a menu item)
+ * Modifier: defines the appearance and behavior of a block or an element
+
+BEM methodology solves common frontend problems:
+ 1. Component approach (splitting interface into blocks).
+ 2. How to name things (in code and on filesystem) — block, element, modifier [convention](https://en.bem.info/method/naming-convention/).
```

Comment:
2. How to name things (in [code](https://en.bem.info/method/naming-convention/) and on [filesystem](https://en.bem.info/method/filesystem/)).